### PR TITLE
Add basic version of /uptime command

### LINKED ISSUE #27: Add `/uptime` command to display current uptime of bot

### PR DESCRIPTION
**Pull request by koyakonsta**
fix #27

### CODE REVIEW

File: src/extensions/uptime.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import time as t; starttime = t.time()
+import arc
+import hikari
+import datetime as dt
+
+upt_plugin = arc.GatewayPlugin("Blockbot Uptime")
+
+@upt_plugin.include
```
Comment (DistroByte) [inline src/extensions/uptime.py:8]: Code should be readable by design!

```suggestion
@upt_plugin.include
uptime_plugin.include
```
```

File: src/extensions/uptime.py [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+upt_plugin = arc.GatewayPlugin("Blockbot Uptime")
+
+@upt_plugin.include
+@arc.slash_command("uptime", "Show formatted uptime of Blockbot")
+async def uptime(ctx):
+    await ctx.respond(f"Uptime: **{dt.timedelta(seconds=t.time()-starttime)!s}**")
+
+@arc.loader
+def loader(client):
+    client.add_plugin(upt_plugin)
```
Comment (DistroByte) [inline src/extensions/uptime.py:16]:
```suggestion
    client.add_plugin(uptime_plugin)

```

**Review by DistroByte — DISMISSED** [review]
LGTM, please try and keep your commits descriptive though 😂

**Comment by koyakonsta** [comment]
> LGTM, please try and keep your commits descriptive though 😂

DW, I'm getting the hang of this,,, slowly ( •̀ ω •́ )✧😂

File: src/extensions/uptime.py [hunk 3]
```diff
@@ -0,0 +1,15 @@
+import time as t; starttime = t.time()
```
Comment (JedHazaymeh) [inline src/extensions/uptime.py:1]: Can this import be cleaner?
```py
from time import time
# ...
start = time() # naming convention says use start or start_time
```

File: src/extensions/uptime.py [hunk 4]
```diff
@@ -0,0 +1,15 @@
+import time as t; starttime = t.time()
+import arc
+import hikari
+import datetime as dt
+
+uptime_plugin = arc.GatewayPlugin("Blockbot Uptime")
```
Comment (JedHazaymeh) [inline src/extensions/uptime.py:6]: The other plugins use just `plugin` as the name, just to keep consistency across the codebase.

Comment (koyakonsta) [reply src/extensions/uptime.py:6]: sure ¯\(°_o)/¯

File: src/extensions/uptime.py (hunk 3 above)
Comment (koyakonsta) [reply src/extensions/uptime.py:1]: wilco （￣︶￣）

**Comment by JedHazaymeh** [comment]
@koyakonsta Is it possible to move this PR to a branch of `redbrick/blockbot` instead of a fork, so we can QA the changes on the review deployment?

**Comment by koyakonsta** [comment]
> @koyakonsta Is it possible to move this PR to a branch of `redbrick/blockbot` instead of a fork, so we can QA the changes on the review deployment?

will do (⁠☉⁠｡⁠☉⁠)⁠!⁠→

**Pull request closed without merging**